Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -283,7 +282,9 @@ public class SchemeGenerator {
             launchAction: launchAction,
             profileAction: profileAction,
             analyzeAction: analyzeAction,
-            archiveAction: archiveAction
+            archiveAction: archiveAction,
+            wasCreatedForAppExtension: target
+                .flatMap({ $0.type.isExtension ? true : nil })
```

Comment:
`target?.type.isExtension` generates wasCreatedForAppExtension = "NO"

Reply:
I think we should use `schemeTarget`, and super small nitpick can we remove the parenthesis around flatMap 😄